### DISCUSSION TITLE
请求一个帮助，通过nb docker部署后的网络连接问题

### DISCUSSION THREAD

**Discussion by xingjianya-86** [nonebot/discussions#481] [问答] [answered]
我的QQ在另外一个容器里面，请问我如何通过ws反向代理？
nonebot的日志打印如下
07-23 09:52:10 [SUCCESS] nonebot | NoneBot is initializing...
07-23 09:52:10 [INFO] nonebot | Current Env: prod
07-23 09:52:11 [SUCCESS] nonebot | Succeeded to load plugin "echo" from "nonebot.plugins.echo"
07-23 09:52:11 [SUCCESS] nonebot | Running NoneBot...
07-23 09:52:11 [INFO] nonebot | Application startup complete

而QQ这头的日志
2024-07-23 10:00:09 [ERROR] | 反向ws断开 ws://172.17.0.4:8080/onebot/v11/ws/ 
2024-07-23 10:00:09 [INFO]  | {"errno":-111,"code":"ECONNREFUSED","syscall":"connect","address":"172.17.0.4","port":8080} 

无论我设置IP地址是 127.0.0.1 还是docker指定IP都是这样，求助解决办法。

**Comment by yanyongyu** [comment, marked as answer]
你的docker应该映射端口，通过映射端口连接

**Reply by xingjianya-86** [reply]
谢谢您~我解决了。是nontbot2docker的时候没有写入对应的驱动器
